src/header/header.scss

```diff
@@ -252,20 +204,21 @@
       overflow: hidden;
       text-overflow: ellipsis;
     }
+  }
 
-    .iui-description {
-      height: $iui-baseline * 2;
-      @media (prefers-reduced-motion: no-preference) {
-        transition: all $iui-speed-fast ease;
-      }
-      @include themed {
-        opacity: t(iui-opacity-3);
-      }
+  .iui-description {
+    height: $iui-baseline * 2;
+    @media (prefers-reduced-motion: no-preference) {
+      transition: all $iui-speed-fast ease;
+    }
+    @include themed {
+      opacity: t(iui-opacity-3);
     }
   }
 
   > .iui-button-icon {
     flex-shrink: 0;
+    box-sizing: content-box;
```

Comment:
this is getting overriden

Reply:
oh, by the same value defined below..

Reply:
You are correct, removed from here & left on `.iui-header-button-icon`.